platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -170,7 +170,7 @@ public void run() {
 
 	public boolean onTouchEvent(final MotionEvent event) {
 		// Mouse drag (mouse pressed and move) doesn't fire onGenericMotionEvent so this is needed
-		if (event.isFromSource(InputDevice.SOURCE_MOUSE)) {
+		if (event.isFromSource(InputDevice.SOURCE_MOUSE) || event.isFromSource(InputDevice.SOURCE_STYLUS)) {
```

Comment:
This means that now `onGenericMotionEvent` need to do a the work except for ACTION_MOVE, and because you didn't implemented touch there only move works for now